kinopoisk/movie/__init__.py

```diff
@@ -80,32 +80,14 @@ def add_series_season(self, year, episodes):
 class Trailer(object):
     def set_defaults(self):
         self.id = None
-        self.width = None
-        self.heigth = None
         self.file = None
-        self.dom = 'tr'
-        self.advsys = 'rutube'
-        self.sbt = ''
-        self.genres = None
-        self.preview_file = None
-        self.preview_width = None
-        self.preview_heigth = None
 
     def __init__(self, params=None):
         self.set_defaults()
 
         if params:
-            self.id = params['trailerId'].replace('top', '')
-            self.width = params['trailerW']
-            self.heigth = params['trailerH']
-            self.file = params['trailerFile']
-            self.dom = params['trailerDom']
-            self.advsys = params['trailerAdvsys']
-            self.sbt = params['trailerSbt']
-            self.genres = params['genres']
-            self.preview_file = params['previewFile']
-            self.preview_width = params['previewW']
-            self.preview_heigth = params['previewH']
+            self.id = params.get('id')
+            self.file = self.trailer_file
```

Comment:
we don't need this line and line 83 if you rename `trailer_file` property to simple `file`.